ls1intum/Artemis
# Feature/exam mode/exam management

# Checklist

### Motivation and Context
This PR creates the exam management view and the functionality to display exams for courses as well as delete exams. See #1618

### Description
Created a view for the exam management and forwarded the appropriate requests to the server. 
Within the scope of this PR are the listing of the exams and the ability to delete an exam from the server. For all other buttons, @sascha11110 's routes have been used but they link to 'TODO' sites or throw an error if there is no appropriate mapping yet. 

### Steps for Testing

1. Go to https://artemistest.ase.in.tum.de
2. Log in to Artemis
3. Go to Course Management
4. Click on Exams
5. Create an Exam.
6. The Exam should now be visible if you are an admin/instructor
7. Delete a Course, it should be deleted from the server

### Screenshots

- Course Management
![image](https://user-images.githubusercontent.com/15110960/84533207-f1a88f00-ace7-11ea-9653-5fa2b0e84b73.png)

- Exam Management
![image](https://user-images.githubusercontent.com/15110960/84533231-fd945100-ace7-11ea-93c3-770ff730e38b.png)

- Delete Exam 
![image](https://user-images.githubusercontent.com/15110960/84533249-04bb5f00-ace8-11ea-8e0a-d8a45d28f0b8.png)
